core-rust/core-api-server/src/core_api/server.rs

```diff
@@ -142,6 +142,11 @@ pub async fn create_server<F>(
         .route("/state/epoch", post(handle_state_epoch))
         .route("/state/clock", post(handle_state_clock))
         .route("/state/component", post(handle_state_component))
+        .route("/state/validator", post(handle_state_validator))
+        .route(
```

Comment:
Extremely minor, but should this be on a single line, for consistency?

Reply:
So we use autoformatters (cargo format and clippy) which do a good job at generally cleaning things up, but sadly in specific circumstances, they can make things look rubbish or inconsistent. Sadly it looks like this is one of those cases! :)